app/routes/application.js

```diff
@@ -0,0 +1,33 @@
+import Route from '@ember/routing/route';
+import ENV from 'website-my/config/environment';
+
+const API_BASE_URL = ENV.BASE_API_URL;
+export default class ApplicationRoute extends Route {
+  model = async () => {
+    const defaultName = 'User';
+    const defaultPicture = 'dummyProfilePicture.png';
+
+    try {
+      const response = await fetch(`${API_BASE_URL}/users/self`, {
+        credentials: 'include',
+      });
+      const userData = await response.json();
```

Comment:
Sweet! Now all of our pages will be behind auth. Thanks for this one! ⚡

Reply:
Yep 🔥, you're welcome